server/sonar-server/src/main/java/org/sonar/server/computation/component/ComponentImpl.java

```diff
@@ -83,6 +89,27 @@ public int getRef() {
     return component.getRef();
   }
 
+  public String getUuid() {
+    return uuid;
+  }
+
+  public ComponentImpl setUuid(String uuid) {
```

Comment:
too bad the current version is mutable... but in this case, the method from interface is missing a `@CheckForNull`